src/domains/customer.js

```diff
@@ -1,26 +1,37 @@
-const prisma = require('../utils/prisma')
+const prisma = require('../utils/prisma');
 
-/**
- * This will create a Customer AND create a new Contact, then automatically relate them with each other
- * @tutorial https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#create-a-related-record
- */
-const createCustomerDb = async (name, phone, email) => await prisma.customer.create({
-  data: {
-    name,
-    contact: {
-      create: {
-        phone,
-        email
-      }
-    }
-  },
-  // We add an `include` outside of the `data` object to make sure the new contact is returned in the result
-  // This is like doing RETURNING in SQL
-  include: {
-    contact: true
-  }
-})
+const createCustomerDb = async (name, phone, email) => {
+  return await prisma.customer.create({
+    data: {
+      name,
+      contact: {
+        create: {
+          phone,
+          email,
+        },
+      },
+    },
+    include: {
+      contact: true,
+    },
+  });
+};
+
+const updateCustomerDb = async (id, newName) => {
+  return await prisma.customer.update({
+    where: {
+      id: parseInt(id),
+    },
+    data: {
+      name: newName,

```

Comment:
I wonder if these could be shortened even further by using the same names. So instead of having `newName` as the parameter it could be passed through as the same desconstructed key and value like: `name: name`

Then when adding the data all you have to do is: 
`data: {
name,
}`